// hello-world-app/src/components/pages.js
import GreetFunction from './GreetFunction';
import GreetConst from './GreetConst';
import Home from './Home';
import Layout from './Layout';
import WelcomeClass from './WelcomeClass';
import CreateElementUsingJSX from './CreateElementUsingJSX';
import CreateElementUsingConstReturn from './CreateElementUsingConstReturn';
import GreetProps from './GreetProps';
import GreetDestructureProps from './GreetDestructureProps';
import GreetConstDestructureProps from './GreetConstDestructureProps';
import WelcomeClassProps from './WelcomeClassProps';
import MessageClass from './MessageClass';
import CounterClass from './CounterClass';
import CounterClassPropsIncrementFive from './CounterClassPropsIncrementFive';
import CounterClassPrevState from './CounterClassPrevState';
import FunctionClick from './FunctionClick';
import ClassClick from './ClassClick';
import EventBindClass from './EventBindClass';
import EventBindClassArrowFunction from './EventBindClassArrowFunction';
import ParentComponent from './ParentComponent';
import UserGreetingIfElse from './UserGreetingIfElse';
import UserGreetingIfElseSecondMethod from './UserGreetingIfElseSecondMethod';
import UserGreetingIfElseTernaryMethod from './UserGreetingIfElseTernaryMethod';
import UserGreetingIfElseShortCircuit from './UserGreetingIfElseShortCircuit';
import NameList1 from './NameList1';
import NameList2 from './NameList2';
import NameList3 from './NameList3';
import NameListKeyIndex from './NameListKeyIndex';
import Stylesheet from './Stylesheet';
import Inline from './Inline';


export {

    GreetFunction,
    GreetConst,
    WelcomeClass,
    CreateElementUsingJSX,
    CreateElementUsingConstReturn,
    GreetProps,
    GreetDestructureProps,
    GreetConstDestructureProps,
    WelcomeClassProps,
    MessageClass,
    CounterClass,
    CounterClassPropsIncrementFive,
    CounterClassPrevState,
    FunctionClick,
    ClassClick,
    EventBindClass,
    EventBindClassArrowFunction,
    ParentComponent,
    UserGreetingIfElse,
    UserGreetingIfElseSecondMethod,
    UserGreetingIfElseTernaryMethod,
    UserGreetingIfElseShortCircuit,
    NameList1,
    NameList2,
    NameList3,
    NameListKeyIndex,
    Stylesheet,
    Inline,    
    Home,
    Layout

}
